cmd/kubectl-testkube/commands/testworkflows/run.go

```diff
@@ -217,7 +217,13 @@ func watchTestWorkflowLogs(id string, signature []testkube.TestWorkflowSignature
 		// Strip timestamp + space for all new lines in the log
 		for len(l.Log) > 0 {
 			if isLineBeginning {
-				l.Log = l.Log[LogTimestampLength+1:]
+				if len(l.Log) >= 29 && l.Log[29] == '+' {
```

Comment:
should be const

Reply:
I'm not sure - probably it may be less clean with that. I would think of something like `LogTimeFormatDiscriminatorCharIndex = 29` and `LogTimeFormatDiscriminatorTimezone = '+'`, but while it's descriptive, it's quite obfuscating. Do you have some other proposal?

Reply:
At least it will tell you what is 29 about. Or add the comment instead. It's a matter of taste, ofc